catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/managers/ProductMgrImpl.java

```diff
@@ -119,10 +119,9 @@ public GetParametersRequest get_parameters(Product prod, String[] desired_parame
             getParamRequest = GetParametersRequestHelper.narrow(obj);
 
         } catch (WrongAdapter | WrongPolicy | UnsupportedEncodingException e) {
-            LOGGER.error("Unable to get ID from product reference: {}", NsilCorbaExceptionUtil.getExceptionDetails(e));
-            LOGGER.debug("Product reference exception details", e);
+            LOGGER.debug("Unable to get ID from product reference:", e);
```

Comment:
NsilCorbaExceptionUtil.getExceptionDetails is willing to go into the Corba Exception Object  and log the Corba wrapped exception information. If the exception is not one of the Corba exception types it just returns the exception.toString().